metrics/api/src/test/java/io/helidon/metrics/api/TestScopeConfig.java

```diff
@@ -16,27 +16,26 @@
 package io.helidon.metrics.api;
 
 import java.util.Map;
+import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
 import io.helidon.config.Config;
+import io.helidon.config.ConfigMappingException;
 import io.helidon.config.ConfigSources;
 
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
-// TODO enabled once patterns are back
-@Disabled
 class TestScopeConfig {
 
     @Test
     void checkEnableAllPattern() {
         ScopeConfig mts = ScopeConfig.builder()
                 .name("test")
-                .include(".*")
+                .include(Pattern.compile(".*"))
```

Comment:
You could add custom methods to builder also support `String` here if you wanted

Reply:
Added